Components/Optimizers/ConjugateGradient/itkGenericConjugateGradientOptimizer.h

```diff
@@ -111,16 +111,16 @@ class GenericConjugateGradientOptimizer : public ScaledSingleValuedNonLinearOpti
    *   GradientMagnitudeTolerance * \max(1, \|CurrentPosition\| ) \f$
    */
   itkGetConstMacro(GradientMagnitudeTolerance, double);
-  itkSetMacro(GradientMagnitudeTolerance, double)
-
-    /** Setting: a stopping criterion, the value tolerance. By default 1e-5.
-     *
-     * The optimizer stops when
-     * \f[ 2.0 * | f_k - f_{k-1} | \le
-     *   ValueTolerance * ( |f_k| + |f_{k-1}| + 1e-20 ) \f]
-     * is satisfied MaxNrOfItWithoutImprovement times in a row.
-     */
-    itkGetConstMacro(ValueTolerance, double);
+  itkSetMacro(GradientMagnitudeTolerance, double);
+
+  /** Setting: a stopping criterion, the value tolerance. By default 1e-5.
+   *
+   * The optimizer stops when
+   * \f[ 2.0 * | f_k - f_{k-1} | \le
+   *   ValueTolerance * ( |f_k| + |f_{k-1}| + 1e-20 ) \f]
+   * is satisfied MaxNrOfItWithoutImprovement times in a row.
+   */
+  itkGetConstMacro(ValueTolerance, double);
```

Comment:
@Wildcarde FYI, I only just applied your patch, but then the automatic source code formatting (clang-format) also made some adjustments, as you see here!